framework/Makefile

```diff
@@ -52,7 +52,8 @@ install:
 	# ETC
 	$(INSTALL_DIR) $(ETC_INSTALLDIR)
 	$(INSTALL_DIR) $(ETC_INSTALLDIR)/cluster
-	$(INSTALL_DIR) $(ETC_INSTALLDIR)/shared
+	$(INSTALL_DIR) $(ETC_INSTALLDIR)/certs
+	$(INSTALL_DIR) $(ETC_INSTALLDIR)/groups
```

Comment:
Also is needed to modify the [SPEC file](https://github.com/wazuh/wazuh/blob/5245e81db6175631b60d9432d0849a2eac437f4d/packages/rpms/SPECS/wazuh-server.spec#L243) for RPM packages